graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/SortOrderFactory.java

```diff
@@ -0,0 +1,12 @@
+package org.graylog.storage.elasticsearch6;
+
+import org.elasticsearch.search.sort.SortOrder;
+import org.graylog2.indexer.searches.Sorting;
+
+import java.util.Locale;
+
+public class SortOrderFactory {
```

Comment:
How about `SortOrderMapper` to make it sound less pompous? :)

Reply:
You are underestimating the very complicated and demanding job of looking up the corresponding sort order! :)